build-tools/build.gradle

```diff
@@ -65,7 +65,6 @@ def generateVersionProperties = tasks.register("generateVersionProperties", Writ
 
 tasks.named("processResources").configure {
     from(generateVersionProperties)
-    exclude 'buildSrc.marker'
```

Comment:
A damn I was so much looking to remove this when removing buildParams.isInternal... And then I missed it... 🤦